Duplicati/Server/webroot/ngax/scripts/services/EditUriBuiltins.js

```diff
@@ -749,9 +769,19 @@ backupApp.service('EditUriBuiltins', function (AppService, AppUtils, SystemInfo,
             'tardigrade-satellite': scope.tardigrade_satellite,
             'tardigrade-api-key': scope.tardigrade_api_key,
             'tardigrade-secret': scope.tardigrade_secret,
+			'tardigrade-secret-verify': scope.tardigrade_secret_verify,
             'tardigrade-shared-access': scope.tardigrade_shared_access,
 			'tardigrade-bucket': scope.tardigrade_bucket,
-			'tardigrade-folder': scope.tardigrade_folder
+			'tardigrade-folder': scope.tardigrade_folder,
+			
+			'storj-auth-method': scope.storj_auth_method,
+            'storj-satellite': scope.storj_satellite,
+            'storj-api-key': scope.storj_api_key,
+            'storj-secret': scope.storj_secret,
+			'storj-secret-verify': scope.storj_secret_verify,
+            'storj-shared-access': scope.storj_shared_access,
+			'storj-bucket': scope.storj_bucket,
+			'storj-folder': scope.storj_folder
```

Comment:
Can we fix the indentation here (spaces vs. tabs) and above?

Reply:
Fixed